src/python/grpcio/grpc/early_adopter/implementations.py

```diff
@@ -214,6 +219,7 @@ def stub(
       should be used.
     certificate_chain: The PEM-encoded certificate chain to use or None if no
       certificate chain should be used.
+    thread_pool_size: Number of threads to create for the backing thread pool.
```

Comment:
This insertion appears out of order? thread_pool_size appears after server_host_override in the function's arguments, right?